### PR TITLE
Added save image to file option.

### PR DESCRIPTION
**Pull request by javierportillo**
I needed to save the image file for my application, so I added this functionality.
I also added some application identifiers and updated the readme file.
This is my first pull request btw.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 GS1 barcode generator
 =====================
 
-The library generates GS1 barcode. Currently supported formats: GS1 128. Code is under development so it may change
```
Comment (javierportillo) [inline readme.md]: I'm using Atom in Linux and it changed some line endings, sorry for the mess.

File: readme.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 GS1 barcode generator
 =====================
 
-The library generates GS1 barcode. Currently supported formats: GS1 128. Code is under development so it may change 
-radically. If you found an issue just let me know. 
+The library generates GS1 barcode. Currently supported formats: GS1 128. Code is under development so it may change
+radically. If you found an issue just let me know.
```
Comment (javierportillo) [inline readme.md:5]: I'm using Atom in Linux and it changed some line endings in this file. Sorry for the mess.

**Comment by ayeo** [comment]
I am glad you found the library useful and even decided to contribute. I will review the changes and apply them soon. Thank you ;)

**Comment by ayeo** [comment]
Sorry for delay. I gladly add your changes involving new application identifiers. When it comes to saving barcode it gets complicated. The Png class is the Response, it represents http response. If you need to save result as file you need a seperate class doing this. I realized there is mess in Builder, and the class purpose may be a bit misleadnig.

**Comment by javierportillo** [comment]
I moved the functionality to its own class. Tell me what you think.

**Comment by ayeo** [comment]
Thank you for your contribution, I have merged your changes ;)

File: readme.md [hunk 3]
```diff
@@ -17,24 +17,36 @@ This will generate png image using black and white and default font
...
 
- 
+
 Custom barcode:
 
 ```php
 $builder = new Barcode\Builder();
 $builder->setBarcodeType('gs1-128');
 $builder->setFilename('barcode.png');
+$builder->saveImageFile(false);
```
Comment (javierportillo) [inline readme.md:27]: Thanks for merging man. Btw I forgot to remove this line from readme, sorry. :sweat_smile:

**Comment by ayeo** [comment]
Fixed ;)

**Pull request merged**